recipes/chemcat/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "chemcat" %}
+{% set version = "0.3.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/chemcat-{{ version }}.tar.gz
+  sha256: 0d57123d2397100c34afdc522954bf050e17dafcaa69a7c20f8c26f1bc3a87af
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: true  # [win]
+  skip: true  # [py<35]
```

Comment:
This can be removed as conda-forge has a minimum supported version of 3.7
```suggestion
```

Reply:
OK, fixed in the next commit. Thanks for the revision!